development/cpp/configuring_an_ide/visual_studio_code.rst

```diff
@@ -30,20 +30,36 @@ Importing the project
 
 - Within the ``tasks.json`` file find the ``"tasks"`` array and add a new section to it:
 
-.. code-block:: js
-
-  {
-    "label": "build",
-    "type": "shell",
-    "command": "scons",
-    "group": "build",
-    "args": [
-      "platform=linuxbsd", // Change to your current platform
-      "target=debug",
-      "-j4"
-    ],
-    "problemMatcher": "$msCompile"
-  }
+.. tabs::
+  .. code-tab:: js LinuxBSD
+
+    {
+      "label": "build",
+      "group": "build",
+      "type": "shell",
+      "command": "scons",
+      "args": [
+        "platform=linuxbsd",
```

Comment:
The platform argument can be removed since Godot's SCons setup will detect your target platform automatically:

```suggestion
```

Reply:
I guess there is no need for tabs here then, as that's the only difference.

Reply:
Ah, well, with your other suggestion we still need the tabs 🙂